lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -71,21 +71,18 @@ enum ExecutionModel {
     ExecutionModelFragment = 4,
     ExecutionModelGLCompute = 5,
     ExecutionModelKernel = 6,
-    ExecutionModelCount, /* internal use only */
```

Comment:
I opened an issue against spirv header and see if such *Count enum values can be added. Let's wait and see.